api/src/pcapi/use_cases/save_venue_bank_informations.py

```diff
@@ -135,36 +126,20 @@ def execute(self, application_id: str, procedure_id: str | None = None) -> BankI
         else:
             raise NotImplementedError()
 
-        if procedure_id in (
-            settings.DMS_VENUE_PROCEDURE_ID,
-            settings.DMS_VENUE_PROCEDURE_ID_V2,
-            settings.DMS_VENUE_PROCEDURE_ID_V3,
-        ):
+        if procedure_version in (2, 3):
             assert updated_bank_information  # for typing purposes
-            business_unit = BusinessUnit.query.filter(BusinessUnit.siret == siret).one_or_none()
-            # TODO(xordoquy): remove the siret condition once the old DMS procedure is dropped
-            if siret:
-                if not business_unit:
-                    business_unit = BusinessUnit(
-                        name=venue.publicName or venue.name,
-                        siret=siret,
-                        bankAccountId=updated_bank_information.id,
+            if application_details.status == BankInformationStatus.ACCEPTED:
+                try:
+                    offerers_api.link_venue_to_reimbursement_point(venue_sql_entity, venue.identifier)
+                except ApiErrors as exc:
+                    logger.error(
+                        "Could not link venue to itself as its reimbursement point with v3 application",
+                        extra={
+                            "application_id": application_details.application_id,
+                            "err": exc.errors,
+                            "venue": venue_sql_entity.id,
+                        },
```

Comment:
Hmm, c'est moi qui ai ajouté ce message et il est un peu faux. On est dans un bloc qui gère les procédures v2 et v3, donc : 

```suggestion
                        "Could not link venue to itself as its reimbursement point with legacy DMS procedure",
                        extra={
                            "procedure_version": procedure_version,
                            "application_id": application_details.application_id,
                            "err": exc.errors,
                            "venue": venue_sql_entity.id,
                        },
```